pkg/api/customization/monitor/handler_utils.go

```diff
@@ -223,6 +224,9 @@ func parseMetricParams(userContext *config.UserContext, nodeLister v3.NodeLister
 		if err != nil {
 			return newMetricParams, err
 		}
+		if !validateNS(newMetricParams, ns) {
+			return nil, httperror.NewAPIError(httperror.PermissionDenied, "can't access to namespace "+ns+" in this project")
```

Comment:
Shall we log the project name as well? We should also alter the error message slightly. "Can't access namespace <name> from project <>"